core/include/detray/propagator/actors/parameter_resetter.hpp

```diff
@@ -45,7 +45,7 @@ struct parameter_resetter : actor {
             const auto& trf3 = trf_store[surface.transform()];
 
             // Note: How is it possible with "range"???
-            const auto& mask = mask_group[surface.mask_range()];
+            const auto& mask = mask_group[surface.mask().index()];
```

Comment:
Did you do something to make it work?

Reply:
No I just wondered if `mask_range()` and `mask().index()` are different.

Reply:
Unfortunately not, it just reduced the number of methods in the surface class